### PR TITLE
Added Sensor (Lidar and Imu) Test

### PR DESCRIPTION
**Pull request by szylis**
Add tests to check the correctness of LiDAR and IMU sensor.

### CODE REVIEW

File: Assets/Tests/PlayMode/Sensors/SensorsTest.cs [hunk 1]
```diff
@@ -7,11 +7,11 @@
 using UnityEngine.SceneManagement;
 using UnityEngine.TestTools.Utils;
 using AWSIM;
+using AWSIM.Tests;
 
 public class SensorsTest
```
Comment (mackierx111) [inline Assets/Tests/PlayMode/Sensors/SensorsTest.cs:12]: @szylis
Could you add a comment explaining the content of this test?

**Review by mackierx111 — CHANGES_REQUESTED** [review]

File: Assets/Tests/PlayMode/Sensors/SensorsTest.cs (hunk 1 above)
Comment (szylis) [reply Assets/Tests/PlayMode/Sensors/SensorsTest.cs:16]: @mackierx111 Done

**Comment by szylis** [comment]
@Goldob I've updated this branch to main.

**Review by mackierx111 — APPROVED** [review]
LGTM!

**Pull request merged**